### DISCUSSION TITLE
关于识别度

### DISCUSSION THREAD

**Discussion by gulichangan** [hiroi-sora/Umi-OCR#506] [General]
我想了解一下OCR引擎
我有使用过Tesseract-ocr
和umi-ocr的识别准确率还是挺高的，关于umi-ocr的引擎是你们自己打造的吗，应为我对ocr这一块了解还不是很深
![20245743451](https://github.com/hiroi-sora/Umi-OCR/assets/69832006/39a370e4-5a76-428c-af64-7d01994d4f46)
![20245743445](https://github.com/hiroi-sora/Umi-OCR/assets/69832006/5d69883f-7be2-41ff-a5f8-02be410eead5)
对比之下Tesseract-ocr的识别率更高 我想问一下对于识别度这一块umi-ocr还会有更高的提升吗

**Comment by hiroi-sora** [comment]
你好。

1. 并不是自己开发的引擎，而是主要基于 [PaddleOCR](https://github.com/PaddlePaddle/PaddleOCR) 。它是当前中文识别准确率较好的开源引擎。
2. Umi-OCR支持多种开源引擎，支持以插件的形式导入不同引擎，包括你说的TesseractOCR我们都是有相应插件的。可以前往 [Umi-OCR_plugins](https://github.com/hiroi-sora/Umi-OCR_plugins) ，下载/导入该插件。
